modules/ROOT/pages/x12-receive-read-settings.adoc

```diff
@@ -1,167 +1,166 @@
-= X12 Settings: Receive from Partners 
+= Receive from Partners
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
+Refer to these as you set up your X12 settings to receive messages from your partners.
+There are two levels of settings, global and partner.
+Global level settings control all X12 messages received from all partners.
+Partner level settings are optionally configured for each partner and override global settings.
```

Comment:
These four lines will currently appear as one paragraph. If you want them to be four paragraphs, add a blank line between each of them.

Reply:
Nope, I think this is fine.